APPTS 10.3.0 release note (#72)
Axway/platform-management-docs: content/en/docs/release_notes/10_03_00_relnotes.md

---
title: Platform Management 10.3.0 release notes
linkTitle: Platform Management 10.3.0 release notes
weight: 78
date: 2022-04-28
Hide_readingtime: true
---

## Platform Management 10.3.0 - 28 April 2022

Platform Management 10.3.0 is a minor release which includes two new features and several improvements and fixed issues.

> Please note that SMS-base multi-factor authentication has been disabled and will be removed in a future release. If you previously had SMS selected as your preferred MFA method, device authorization codes will instead be sent via email. It is recommended that you configure Authenticator App support on your account [_Credentials_](https://platform.axway.com/user/credentials) view.

Reviewer: ```suggestion
Please note that SMS-base multi-factor authentication has been disabled and will be removed in a future release. If you previously had SMS selected as your preferred MFA method, device authorization codes will be sent by email instead. It is recommended that you configure Authenticator App support on your account [_Credentials_](https://platform.axway.com/user/credentials) view.
```

## New features

> * Users who are members of multiple organizations and wish to use their Identity Provider to provision and authenticate users in those orgs can now configure their email domain(s) and associate them to a shared Identity Provider configuration. Please see Enabling Identity Provider for multiple orgs in the [Configuring and managing Identity Providers documentation](https://docs.axway.com/bundle/platform-management/page/docs/management_guide/configuring_and_managing_identity_providers/index.html) for more information.

Reviewer: ```suggestion
* Users who are members of multiple organizations and wish to use their Identity Provider to provision and authenticate users in those orgs can now configure their email domain(s) and associate them to a shared Identity Provider configuration. See [Enabling an Identity Provider for multiple organizations](/docs/management_guide/configuring_and_managing_identity_providers/enabling_identity_provider_multiple_orgs) for more information.
```

For the first sentence, my suggestion is that you write it in two sentences by first saying was these users were able to do previously ("Previously, users who are members of multiple organizations could do such and such"). Then the next sentence could say "Now they can do such and such." No need to say anything about they wish or want... just say they can now do it (it's implied if they want to do it or that it's an additional option). Since "initial" organization and "additional" organization(s) is used in the docs, do you want to use that terminology - see what is written in that new page to see if that can help to shorten the description?
